Makefile

```diff
@@ -188,7 +188,7 @@ dev-down-aws: build-installer
 .PHONY: dev-run-gcp
 # Controller manager configuration #
 dev-run-gcp: export CLOUD=gcp
-dev-run-gcp: PROJECT_ID ?= $(shell gcloud config get project)
```

Comment:
I purposely used `?=` so it doesn't get rerun every time. Any reason why you had to change it back to `export`?

Reply:
seems it causes the project ID to be not set correct so that's why you changed it. Lets use what you got for now and I can figure out another way to only have to run `gcloud config get project` once.